meetings/2020-03-03.md

```diff
@@ -0,0 +1,178 @@
+# OpenJS Foundation Cross Project Council Meeting 2020-03-03
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=ChAmBmGmKIQ
+* **GitHub Issue**: https://github.com/openjs-foundation/cross-project-council/issues/477
+
+## Present
+
+* Joe Sepi (@joesepi)
+* Kris Borchers (@kborchers)
+* Ben Michel (@obensource)
+* MylesBorins(@MylesBorins)
+* Darcy Clarke (@darcyclarke)
+* Jordan Harband (@ljharb)
+* Christopher Hiller (@boneskull)
+* Michael Dawson (@mhdawson)
+* Rachel Romoff (@rromoff)
+* Eemeli Aro (@eemeli)
+* Robin Ginn (@rginn)
+* Tierney Cyren (@bnb)
+
+## Agenda
+
+### Announcements
+
+Myles: Tickets are available for OpenJS world!
+
+Rachel: Webhint AMA tomorrow at 9am PT – please join!
+We’re trying 9am now and seeing how the timing goes, we’re open to feedback though. As long as timing doesn’t overlap with an OpenJS meeting, the timing will be good.
+We’re trying to accommodate as many folks as possible.
+
+### Board Meeting Updates
+
+Michael: We had the board meeting this week, hoping that Brian will have something to share on the projects for Electron / AMP who’re able to contribute to receive directed funding.
+
+Myles: We met last Friday, and we have an in-person meeting this Monday. If there’s anything you’d like us to consider or discuss before then, please let us know since we’ll be moving fast on Monday.
+
+Michael: The public portion of the meeting on Monday will be at 3pm–5pm PST / 6pm–8pm ET.
```

Comment:
Michael: We agreed to do the public session at the start of the next meeting on Monday March 9th at 3pm PST / 6pm ET.